### PR TITLE
Removing broken Travis webhook integration

### PR DESCRIPTION
**Pull request by Valloric**
- Fixed incorrect instructions in the readme.
- Added more explanations and example config for Travis & Appveyor,
which most users will likely need.

Fixes servo/homu#60.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/homu/117)

### CODE REVIEW

**Comment by Valloric** [comment]
cc @Manishearth

**Comment by Valloric** [comment]
cc @cbrewster @edunham @KiChjang @aneeshusa

**Comment by Manishearth** [comment]
unsubmitted review, sorry.

r=me with the venv bits fixed

**Comment by edunham** [comment]
ping @Valloric to add venv bits to readme so we can merge this :)

File: README.md [hunk 1]
```diff
@@ -62,74 +64,67 @@ support.
...
-# Stable version
-
-pip install homu
-
-# Development version
-
-git clone https://github.com/barosl/homu.git
-pip install -e homu
+$ git clone https://github.com/barosl/homu.git
+$ pip install -e homu
```
Comment (Valloric) [inline README.md]: I've never used venv so I'm not confident I can write docs explaining to people how they should use it.

Could that be done as a separate PR though? The current PR only reformats the code that's already there, no?

**Comment by Valloric** [comment]
Didn't see comments were added, apologies for lack of response!

File: README.md (hunk 1 above)
Comment (Manishearth) [inline README.md]: The venv setup was already there in the README, this pull request removes it. Which we probably don't want.

File: README.md [hunk 2]
```diff
@@ -62,74 +64,67 @@ support.
...
-pyvenv .venv
-. .venv/bin/activate
-
-# Stable version
-
-pip install homu
-
-# Development version
-
-git clone https://github.com/barosl/homu.git
```
Comment (Manishearth) [inline README.md]: while you're at it make this servo instead of barosl

File: README.md (hunk 1 above)
Comment (Valloric) [inline README.md]: Sorry, you're right, I forgot what the original code was doing.

Comment (Valloric) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (Valloric) [inline README.md]: Done.

**Comment by Valloric** [comment]
Fixed comments and rebased. Again, apologies for the delay!

**Pull request merged**